src/trans/external/setup_trans.F90

```diff
@@ -10,7 +10,7 @@
 
 SUBROUTINE SETUP_TRANS(KSMAX,KDGL,KDLON,KLOEN,LDSPLIT,PSTRET,&
 &KTMAX,KRESOL,PWEIGHT,LDGRIDONLY,LDUSERPNM,LDKEEPRPNM,LDUSEFLT,&
-&LDSPSETUPONLY,LDPNMONLY,LDUSEFFTW,&
```

Comment:
As mentioned in PR comments, please leave LDUSEFFTW for the moment but print deprecation message when `present(LDUSEFFTW)`.